src/css/reviewPage.css

```diff
@@ -0,0 +1,18 @@
+/* Review Page */
+.review-page__page-heading__text {
+  color: var(--color-primary-darker);
+}
+
+.review-page__page-heading__link {
+  text-align: right;
```

Comment:
Can use `.vads-u-text-align--right` potentially. Does it make sense to use these utility classes in this case, or does it not matter?